app/views/materials/index.html.erb

```diff
@@ -7,7 +7,7 @@
     <% @materials.each do |material| %>
       <li class="list-group-item d-flex justify-content-between align-items-center">
         <div class="d-flex align-items-center justify-content-between">
-          <%= cl_image_tag, height: 20 , width: 20, crop: :fill %>
+          <%#= cl_image_tag, mheight: 20 , width: 20, crop: :fill %>
```

Comment:
I think you need the attachement --> `cl_image_tag, **metarial.photo.key**` for it to work

Reply:
see next commit